### PR TITLE
EOS-27645: Change CORTX Constant for RGW

### PR DESCRIPTION
**Pull request by kailashyadav11**
Signed-off-by: Ujjwal Lanjewar <ujjwal.lanjewar@seagate.com>

# Problem Statement
- s3 service related constants to be replaced by rgw instead

# Design
- (https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/651132980/CORTX+GConf+Config+Store+Keys) s3 services replaced by rgw services instead.

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:   

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/utils/cortx/const.py [hunk 1]
```diff
@@ -24,27 +24,29 @@ class Const(Enum):
...
-    SERVICE_S3_SERVER       = 's3server'
-    SERVICE_S3_HAPROXY      = 'haproxy'
-    SERVICE_S3_AUTHSERVER   = 'authserver'
-    SERVICE_S3_BGWORKER     = 'bgworker'
-    SERVICE_S3_BGSCHEDULER  = 'bgscheduler'
-    SERVICE_CSM_AGENT       = 'agent'
-    SERVICE_UTILS_MESSAGE   = 'message'
-    SERVICE_CCLIENT         = 'cclient_service'
+    SERVICE_MOTR_IO        = 'io'
+    SERVICE_MOTR_FSM       = 'fsm'
```
Comment (ujjwalpl) [inline py-utils/src/utils/cortx/const.py:33]: This "fsm", can be moved to deprecated.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

**Review by kupranay — APPROVED** [review]
looks good

**Review by ArchanaLimaye — APPROVED** [review]
No changes from HA perspective

File: py-utils/src/utils/cortx/const.py (hunk 1 above)
Comment (kailashyadav11) [reply py-utils/src/utils/cortx/const.py]: moved it under deprecated

**Review by ujjwalpl — APPROVED** [review]

**Comment by sachinpunadikar** [comment]
@kailashyadav11  please fill in PR template

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by rahul-tripathi-git — APPROVED** [review]
+1

**Comment by kailashyadav11** [comment]
> @kailashyadav11 please fill in PR template

@sachinpunadikar added basic details in the template please check.

**Pull request merged**